doc/ansible/dev/retroactively-owned-resources.md

```diff
@@ -0,0 +1,146 @@
+# Owner References for Existing Resources
+
+Owner references are automatically injected *only during creation of
+resources*. Enabling owner reference injection *will not update objects*
+created while [owner reference injection is
+disabled](https://github.com/operator-framework/operator-sdk/blob/master/doc/ansible/dev/advanced_options.md#turning-off-dependent-watches-and-owner-reference-injection)
+.
+
+This guide will demonstrate how to retroactively set owner references
+for existing resources.
+
+### Owner References and Annotations
+
+Dependent resources *within the same namespace as the owning CR* are
+tracked with the `ownerReference` field.
+
+```yaml
+
+metadata:
+  ...(snip)
+  ownerReferences:
+    - apiVersion: cache.example.com/v1alpha1
+      kind: Memcached
+      name: example-memcached
+      uid: ad834522-d9a5-4841-beac-991ff3798c00
+```
+
+An `annotation` is used instead of an `ownerReference` if the dependent
+resource is in a different namespace than the CR or the dependent
+resource is not a namespaced resource.
+
+```yaml
+metadata:
+  annotations:
+    operator-sdk/primary-resource: default/example-memcached
+    operator-sdk/primary-resource-type: Memcached.cache.example.com
+```
+
+A GET request to the owning resource will provide the necessary data to
+construct an `ownerReference` or an `annotation`.
```

Comment:
Can you add a small section that just explains exactly how to form the owner reference and annotation, ie that `operator-sdk/primary-resource-type` is `{Kind}.{Group}`, using the kind/group from their Custom Resource, just to really hand-hold. They can figure it out from the examples above but I think going one step more explicit could help.